dap/src/requests.rs

```diff
@@ -169,8 +169,8 @@ pub struct AttachRequestArguments {
 //// Union of Attach and Launch arguments for the Restart request.
 //// Currently the same as LaunchRequestArguments but might not be in the future.
 #[derive(Deserialize, Debug, Default, Clone)]
-#[serde(rename_all(deserialize = "camelCase", serialize = "snake_case"))]
 #[cfg_attr(feature = "client", derive(Serialize))]
+#[serde(rename_all = "camelCase")]
```

Comment:
This `snake_case` was introduced in this PR:
https://github.com/sztomi/dap-rs/pull/26

Which was opened _after_ but merged _before_ this PR, which fixed up all the other uses of `snake_case`:
https://github.com/sztomi/dap-rs/pull/21